src/views/app/commands/index.js

```diff
@@ -0,0 +1,165 @@
+import React, { Component } from 'react';
+import { Row, Col, Form, FormGroup, Label, Input, Button } from 'reactstrap';
+import { Colxx, Separator } from '../../../components/common/CustomBootstrap';
+import Breadcrumb from '../../../containers/navs/Breadcrumb';
+import { getHost } from '../../../helpers/Utils';
+
+export default class Commands extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hftChecked: false,
+      commandType: this.commands[0].command,
+      showResult: false,
+      commandResult: null,
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+}
+
+  commands = [
+    {
+      command: 'server_status',
+    },
+    {
+      command: 'get_securities',
+    },
+    {
+      command: 'get_portfolio',
+    },
+    {
+      command: 'get_forts_positions',
+    },
+    {
+      command: 'gethistorydata',
+    },
+    {
+      command: 'neworder',
+    },
+    {
+      command: 'newstoporder',
+    },
+    {
+      command: 'newcondorder',
+    },
+    {
+      command: 'cancelorder',
+    },
+  ];
```

Comment:
```
commands = [
    'server_status',
    'get_securities',
    ...etc...
].map(command => { return { command }; });
```

Reply:
Moved the commands object to a separate file in constant folder. Planning to keep the command as an object in an array to enable conditional rendering of fields.